src/components/Tokens/TokenDetails/TokenDescription.tsx

```diff
@@ -165,6 +172,37 @@ export function TokenDescription({
           </TruncateDescriptionButton>
         )}
       </TokenDescriptionContainer>
+      {hasFee && (
+        <MouseoverTooltip
+          placement="left"
+          size={TooltipSize.Small}
+          text={
+            <ThemedText.Caption color="neutral2">
+              <FOTTooltipContent />
+            </ThemedText.Caption>
+          }
+        >
+          <Column gap="sm">
+            {sameFee ? (
+              <ThemedText.BodyPrimary>
+                {tokenQuery?.token?.symbol}&nbsp;
+                <Trans>fee:</Trans>&nbsp;{sellFeeString}
+              </ThemedText.BodyPrimary>
+            ) : (
+              <>
+                <ThemedText.BodyPrimary>
```

Comment:
any reason not to combine these two ThemedText.BodyPrimary?

Reply:
Yes, this was to keep them as two separate components for the sake of Column vertical spacing